src/renderers/webvr/WebXRManager.js

```diff
@@ -215,18 +241,18 @@ function WebXRManager( renderer ) {
 
 	function onAnimationFrame( time, frame ) {
 
-		pose = frame.getDevicePose( frameOfReference );
+		pose = frame.getDevicePose !== undefined ? frame.getDevicePose( frameOfReference ) : frame.getViewerPose( frameOfReference );
 
 		if ( pose !== null ) {
 
 			var layer = session.baseLayer;
-			var views = frame.views;
+			var views = frame.views || pose.views;
```

Comment:
TIL XRViewerPose has added `views` and XRView has added `viewMatrix` earlier this week

Reply:
FYI, looks like `viewMatrix` is getting removed again, instead the spec is adding an `.inverse()` method (or `.inverse` attribute?) to the pose XRRigidTransform: 

https://github.com/immersive-web/webxr/pull/560/files